3rdparty/wxwidgets3.0/src/common/sizer.cpp

```diff
@@ -1040,10 +1040,10 @@ wxSize wxSizer::GetMinSize()
     return ret;
 }
 
-void wxSizer::DoSetMinSize( int width, int height )
+void wxSizer::DoSetMinSize(int width, int height)
 {
-    m_minSize.x = width;
-    m_minSize.y = height;
+	m_minSize.x = width;
+	m_minSize.y = height * 1.6; // Hackfix: There is no real fixed height, workaround for general settings where BIOS / Folders / GS Window / Speedhacks and others reside in.
```

Comment:
no no no NO. no 3rdparty patches please damnit

Reply:
Look at this problem: https://github.com/PCSX2/pcsx2/pull/4580

Reply:
Buttons like ok and cancel are dpi sensitive and restore restore defaults just dissapears. It's also bad on other languag

Reply:
see how we handle multi-DPI scaling here: https://github.com/PCSX2/pcsx2/blob/master/pcsx2/gui/Dialogs/AboutBoxDialog.cpp#L38

also either way the bug isn't the issue here, the solution you're proposing is the issue! there is approximately 0% of chance seeing your pr merged with a 3rdparty hack

Reply:
I'll fix it then.